src/resources/search/schema/http-requests.json

```diff
@@ -0,0 +1,26 @@
+{
+  "settings": {
```

Comment:
Having not touched ES for a while - should shards/replicas be hard coded?

Reply:
I was asked to use `json` schema file as reference from `Subscription` hence it is how it is. I can happily remove it.